daphne/cli.py

```diff
@@ -132,6 +135,25 @@ def __init__(self):
             default=False,
             action="store_true",
         )
+        self._arg_proxy_host = self.parser.add_argument(
```

Comment:
I don't like these being stored on `self`, I don't think they're needed later

Reply:
The other way is to pass around within return values and parameters. Since the server class is initialized only once per run, I don't see why storing the parameter result for later use is a problem.